src/app/about/page.js

```diff
@@ -14,7 +14,12 @@ const page = () => {
         }
       />
       <Photo />
-      <Description />
+      <Description
+        title={"Purpose of  Organization"}
+        text={
```

Comment:
Thanks for putting in the text, but this is someone else's issue. You can revert your changes to how it was before